docs/7.x/pack.md

```diff
@@ -1117,3 +1117,21 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+### Kubenetes Admission Controllers
+
+Optional Kubernetes admission controllers can be enabled when using a custom `planet`
```

Comment:
nitpick: I would link to custom system container, and use the same nomenclature as elsewhere in the docs which I think call planet the system container.

Reply:
Yeah, that's probably a good idea to keep it consistent. I'll replaces references to planet to system container.